packages/webapp/src/containers/Finances/AddSale/index.jsx

```diff
@@ -51,7 +55,7 @@ function AddSale() {
     <HookFormPersistProvider>
       <GeneralRevenue
         onSubmit={onSubmit}
-        title={t('common:ADD_ITEM', { itemName: revenueType?.revenue_name })}
+        title={t('common:ADD_ITEM', { itemName: translatedRevenueName })}
```

Comment:
Ooops, how long did we have that bug in there 😆

Reply:
🙏  thanks! And no worries I might've done it too ahah I forget. I noticed it in the edit values which shouldn't really be able to see defaults but they can.